common-compose.yml

```diff
@@ -0,0 +1,70 @@
+wordpress:
+  build: .
+  privileged: true
+  env_file: _env
+  environment:
+    - CONTAINERBUDDY=file:///opt/containerbuddy/config.json
+
+nfs:
+    image: autopilot-nfs
```

Comment:
Is this different from the `misterbisson/nfsserver` from https://github.com/autopilotpattern/nfsserver/pull/1?
(previously in https://github.com/10up/triton-wordpress/pull/2#discussion_r58461022)